### DISCUSSION TITLE
Implementing Auto for one of Intuit's repositories using Jenkins UI

### DISCUSSION THREAD

**Discussion by jinbarai** [intuit/auto#1952] [Q&A]
Hi Andrew,
I'm setting up auto for one of Intuit's repositories using the jenkins UI since we don't have a jenkins file yet. I added a small script as an 'Execute shell' in jenkins. However, I keep getting this error. Any ideas on how to go about it? 

![Screen Shot 2021-04-19 at 11 12 14 AM](https://user-images.githubusercontent.com/43502558/115259709-1b98c200-a100-11eb-98b8-9e9f3512f786.png)

For your reference earlier we had these configurations:
![Screen Shot 2021-04-15 at 12 42 51 PM](https://user-images.githubusercontent.com/43502558/115262733-ce6a1f80-a102-11eb-8de2-0a914ee0ecd1.png)

I changed it to: 
![Screen Shot 2021-04-19 at 11 30 08 AM](https://user-images.githubusercontent.com/43502558/115262766-d4f89700-a102-11eb-82a7-91112f2ec0c1.png)

**Comment by hipstersmoothie** [comment]
It looks like you're trying to make a release for a tag that's already been released
